secrets/secrets.go

```diff
@@ -155,7 +155,7 @@ func (k *Keeper) Close() error {
 	if prev {
 		return errClosed
 	}
-	return k.k.Close()
+	return wrapError(k, k.k.Close())
```

Comment:
wrapError needs to check `err != nil`

Reply:
I think it ended up doing so via gcerrs.DoNotWrap, but better to be explicit. Done.